packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -0,0 +1,235 @@
+import React from 'react';
+import styled from 'styled-components';
+import { StyledActionListItem } from './StyledActionListItem';
+import { componentIds } from './componentIds';
+import type { StyledActionListItemProps } from './getBaseActionListItemStyles';
+import Box from '~components/Box';
+import type { IconComponent } from '~components/Icons';
+import { useDropdown } from '~components/Dropdown/useDropdown';
+import { Text } from '~components/Typography';
+import { isReactNative, makeAccessible, makeSize } from '~utils';
+import { BaseText } from '~components/Typography/BaseText';
+import { Checkbox } from '~components/Checkbox';
+
+const ActionListItemContext = React.createContext<{
+  intent?: ActionListItemProps['intent'];
+}>({});
+
+const SectionDivider = styled(Box)((props) => ({
+  height: makeSize(1),
```

Comment:
Let's add a TODO comment here, if we even add a 1px token we can replace this.

Reply:
done in upcoming commit